docker/yoti/openapi.yml

```
@@ -0,0 +1,640 @@
+          properties:
+            first_name:
+              type: string
+            last_name:
+              type: string
+            email:
+              type: string
+              format: email
+        documents:
+          type: array
+          items:
+            type: object
+            properties:
+              requirement_id:
+                type: string
+              document:
+                type: object
+                properties:
+                  type:
+                    type: string
+                  country_code:
+                    type: string
+                  document_type:
+                    type: string
+
+        branch:
+          type: object
+          properties:
+            type:
+              type: string
+            fad_code:
+              type: integer
+
+    LetterPayload400:
+      type: object
+      properties:
+        code:
+          type: string
+        message:
+          type: string
+        errors:
+          type: array
+          items:
+            type: object
+            properties:
+              property:
+                type: string
+              message:
+                type: string
+
+
+    YotiSession:
+      type: object
+      properties:
+        session_deadline:
+          type: string
+          format: date-time
+        resources_ttl:
+          type: integer
+        ibv_options:
+          type: object
+          properties:
+            support:
+              type: string
+              example: "MANDATORY"
+        user_tracking_id:
+          type: string
+        requested_checks:
+          type: array
+          items:
+            type: object
+            properties:
+              type:
+                type: string
+              config:
+                type: object
+                properties:
+                  manual_check:
+                    type: string
+                  scheme:
+                    type: string
+        required_documents:
+          type: array
+          items:
+            oneOf:
+              - $ref: "#/components/schemas/ID_DOCUMENT"
+              - $ref: "#/components/schemas/SUPPLEMENTARY_DOCUMENT"
+
+        resources:
+          type: object
+          properties:
+            applicant_profile:
+              type: object
+              properties:
+                full_name:
+                  type: string
+                date_of_birth:
+                  type: string
+                  format: dob
+                structured_postal_address:
+                  oneOf:
+                    - $ref: "#/components/schemas/AddressFormat1"
+                    - $ref: "#/components/schemas/AddressFormat2"
+                    - $ref: "#/components/schemas/AddressFormat3"
+                    - $ref: "#/components/schemas/AddressFormat4"
+
+    ID_DOCUMENT:
+      type: object
+      properties:
+        type:
+          type: string
+          enum: [ ID_DOCUMENT, SUPPLEMENTARY_DOCUMENT ]
```

ID_DOCUMENTs only have one option for type
```suggestion
          enum: [ ID_DOCUMENT ]
```